docs/deploying-a-database.adoc

```diff
@@ -0,0 +1,242 @@
+? Enter a value for string property MONGODB_DATABASE (MongoDB Database Name): sampledb
+? Enter a value for string property MONGODB_VERSION (Version of MongoDB Image): 3.2
+? Enter a value for string property VOLUME_CAPACITY (Volume Capacity): 1Gi
+? Provide values for non-required properties No
+? How should we name your service  mongodb-persistent
+? Output the non-interactive version of the selected options No
+? Wait for the service to be ready No
+ ✓  Creating service [32ms]
+ ✓  Service 'mongodb-persistent' was created
+Progress of the provisioning will not be reported and might take a long time.
+You can see the current status by executing 'odo service list'
+----
+
+
+_or_
+
+Deploying the database manually:
+
+. First, let's check what services are avaiable to us:
++
+----
+$ odo catalog list services
+NAME                         PLANS
+django-psql-persistent       default
+jenkins-ephemeral            default
+jenkins-pipeline-example     default
+mariadb-persistent           default
+mongodb-persistent           default
+mysql-persistent             default
+nodejs-mongo-persistent      default
+postgresql-persistent        default
+rails-pgsql-persistent       default
+----
+
+. Next, we will pick `mongodb-persistent` and check what parameters are required to be passed
++
+----
+$ odo catalog describe service mongodb-persistent
+  ***********************        | *****************************************************
+  Name                           | default
+  -----------------              | -----------------
+  Display Name                   |
+  -----------------              | -----------------
+  Short Description              | Default plan
+  -----------------              | -----------------
+  Required Params without a      |
+  default value                  |
+  -----------------              | -----------------
+  Required Params with a default | DATABASE_SERVICE_NAME
+  value                          | (default: 'mongodb'),
+                                 | MEMORY_LIMIT (default:
+                                 | '512Mi'), MONGODB_VERSION
+                                 | (default: '3.2'),
+                                 | MONGODB_DATABASE (default:
+                                 | 'sampledb'), VOLUME_CAPACITY
+                                 | (default: '1Gi')
+  -----------------              | -----------------
+  Optional Params                | MONGODB_ADMIN_PASSWORD,
+                                 | NAMESPACE, MONGODB_PASSWORD,
+                                 | MONGODB_USER
+----
+
+. Pass in the required parameters and wait for the database to be deployed
++
+----
+$ odo service create mongodb-persistent --plan default --wait -p DATABASE_SERVICE_NAME=mongodb -p MEMORY_LIMIT=512Mi -p MONGODB_DATABASE=sampledb -p VOLUME_CAPACITY=1Gi
+----
+
+
+= Connecting the database to the frontend service
+
+. Let's link the database to the frontend service:
++
+----
+$ odo link mongodb-persistent
```

Comment:
Is it worth mentioning that this command needs to be executed from the component directory of the nodejs component created earlier in the document?